awspub/tests/fixtures/config1.yaml

```diff
@@ -82,7 +82,7 @@ awspub:
       marketplace:
         entity_id: "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
         access_role_arn: "arn:aws:iam::xxxxxxxxxxxx:role/AWSMarketplaceAccess"
-        version_title: "1.0.0"
+        version_title: "1.0.0 (arm)"
```

Comment:
Just want to verify that ` (arm)` is intentional here. Does it refer to arch? Because isn't m5.large strictly amd64? Excuse my ignorance on this matter.

Reply:
It is, I just needed something in the title to test against (and ` (arm)` was the literal failure that spawned this. To avoid confusion I'll just update the title to be something like `1.0.0 (testing)`